apps/proto/lib/mix/tasks/namespace/release.ex

```diff
@@ -39,12 +47,19 @@ defmodule Mix.Tasks.Namespace.Release do
 
   defp update_file_contents(path, app_name) do
     contents = File.read!(path)
+    updated_contents = update_script_contents(contents, app_name)
+    File.write!(path, updated_contents)
+  end
+
+  defp update_script_contents(contents, app_name) do
     # matches if preceding characters involves either of: , " [ { [:blank:]
     # this way it doesn't match on substrings or directory names
-    updated_contents =
-      String.replace(contents, ~r/([,"\[{[:blank:]])#{app_name}/, "\\1lx_#{app_name}")
-
-    File.write!(path, updated_contents)
+    contents
+    |> String.replace(~r/([,"\[{[:blank:]])#{app_name}/, "\\1lx_#{app_name}")
+    # Some deps app like `lexical_plugin` have `Elixir.Lexical` in their `app_file`
+    |> String.replace("Elixir.Lexical", "Elixir.LXRelease")
+    |> String.replace("Elixir.Sourceror", "Elixir.LXSourceror")
+    |> String.replace("Elixir.PathGlob", "Elixir.LXPathGlob")
```

Comment:
I think we need to think about this a little more.
Perhaps if we had a mapping from  app ->  root module we could make this a little more extensible.